src/logistration/tests/AccountActivationMessage.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Instead of snapshot testing, can we simply test each case in `LoginPage.test.jsx` like you are doing for `info`?

Reply:
should I just remove these tests?

Reply:
Yes, those tests will cover each scenario.

Reply:
Or, if you want to test it here then instead of snapshots verify the actual message for each case.

Reply:
I have kept the file and verified the actual messages